src/System.Text.Json/src/System/Text/Json/Serialization/JsonExtensionDataAttribute.cs

```diff
@@ -10,6 +10,13 @@ namespace System.Text.Json.Serialization
     /// </summary>
     /// <remarks>
     /// The TKey value must be <see cref="string"/> and TValue must be <see cref="JsonElement"/> or <see cref="object"/>.
+    /// 
+    /// During deserializing, when using <see cref="object"/> a "null" JSON value is treated as a null object reference, and when using
```

Comment:
> `"null"`

You can also wrap this in `<c>null</c>` to indicate a code literal.

Reply:
We probably want to do that for `null` in `... as a null object reference...`, but no the one in quotes. That is referring to a JSON null literal (and not .NET `null` keyword).

Reply:
Kept "null"  for JSON, now using `<c>null</c>` for the null reference.